vsintegration/deployment/WDExpressDeployTemplates/extension.vsixlangpack

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageLanguagePackManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011">
 	<Metadata>
-		<Identity Id="Microsoft.VisualStudio.WdExpressDeployment" Version="14.0" Language="neutral" Publisher="Microsoft" />
+		<Identity Id="Microsoft.VisualStudio.WdExpressDeployment" Version="15.0" Language="neutral" Publisher="Microsoft" />
```

Comment:
Will this install in a vs2015 or are we going to require Dev 15 preview.  I know there are some community members who prefer to avoid preview VS installs.